LEAF_Request_Portal/form.php

```diff
@@ -1392,16 +1400,21 @@ public function hasWriteAccess($recordID, $categoryID = 0, $indicatorID = 0)
             && $this->login->getUserID() == $resRecords[0]['userID'])
         {
             $this->cache["hasWriteAccess_{$recordID}_{$categoryID}"] = 1;
+            $this->log["write_{$recordID}_{$categoryID}_writable"] = "User is a writable user or creator of record {$recordID}, {$categoryID}.";
 
             return 1;
         }
+        $this->log["write_{$recordID}_{$categoryID}_writable"] = "User is not a writable user or creator of record {$recordID}, {$categoryID}.";
```

Comment:
I would recommend changing the language here from "creator" to "initiator" to be consistent with the language across the app.